static/app/utils/discover/fieldRenderers.tsx

```diff
@@ -819,6 +819,10 @@ export function getSortField(
     return field;
   }
 
+  if (field.startsWith('division_if(')) {
```

Comment:
This feels like a hack 🙈, an alternative would be adding them to aggregations, but then we'd need to support this for the discover as well (see count_if as an example, https://github.com/getsentry/sentry/blob/d10148cf0bc3bf203ba0781378956925cef68f4f/static/app/utils/discover/fields.tsx#L291-L318) - not too sure if this would blow stuff up, as the backend support for division_if is tightly bound to releases.

Reply:
I think there's a cleaner way to get around this, but I have to do a bit of investigation. I'm seeing some odd access paths when it comes to the structure of `tableMeta`. If you want, we can remove this until I get a chance to poke around a bit more and we can put up a separate PR to handle the sortability of the headers.

The endpoint returns in the meta data that the corresponding type to the `division_if` statements _should_ be percentages so they should be sortable by default when it gets to [this block](https://github.com/getsentry/sentry/pull/74523/files#diff-04dbefa42672572db5ea85f1a074f5eaf3c6df7703c0e73dfa88747a84335e18R833). I know that the key gets kind of mangled for equations but we should be able to work around that